### DISCUSSION TITLE
Use both SD cards for storing roms in RG351V.

### DISCUSSION THREAD

**Discussion by egcarrano** [351ELEC/351ELEC#640] [Ideas]
### Is your feature request related to a problem? Please describe.

It is not a bug. It is, in fact, a feature request. Would it be possible to use both SD cards on 351V for storing roms? For instance, I would like to use two 128GB SD cards. The first one would store ArkOS itself and some systems; the second one would store other systems.

### What device are you using?
RG351V

**Comment by dhwz** [comment]
If you manually mount your sd that will maybe work, but nothing we will support in the near future.
